packages/tasks/src/model-libraries.ts

```diff
@@ -150,6 +150,18 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"adapter_config.json"`,
 	},
+	deepforest_tree: {
+		prettyLabel: "DeepForest Tree",
+		repoName: "deepforest-Tree",
+		repoUrl: "https://github.com/weecology/DeepForest",
+		countDownloads: `path:"NEON.pt"`,
+	},
+	deepforest_bird: {
+		prettyLabel: "DeepForest Bird",
+		repoName: "deepforest-bird",
+		repoUrl: "https://github.com/weecology/DeepForest",
+		countDownloads: `path:"bird.pt"`,
```

Comment:
```suggestion
		countDownloads: `path_extension:"pt"`,
```

This way as long as there is a `.pt` file - the downloads should show up.